connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerSourceTaskTest.java

```diff
@@ -433,35 +400,36 @@ public void testFailureInPollAfterCancel() throws Exception {
         taskFuture.get();
         assertPollMetrics(0);
 
-        PowerMock.verifyAll();
+        verifyCleanStartup();
+        verify(producer).close(Duration.ZERO);
+        verify(producer).close(Duration.ofSeconds(30));
+        verify(sourceTask).stop();
+        verify(admin).close(any(Duration.class));
+        verify(transformationChain).close();
+        verify(offsetReader, times(2)).close();
```

Comment:
Since `OffsetStorageReaderImpl::close` is idempotent, it doesn't hurt that we invoke it twice, but it's definitely not necessary. We just have to make sure it's invoked at least once:
```suggestion
        verify(offsetReader, atLeast(1)).close();
```